### ISSUE TITLE
Performance on Test Data

### ISSUE THREAD

**Issue by sk210892** [thomasmesnard/DeepMind-Teaching-Machines-to-Read-and-Comprehend#4]
Hi,
How can I check the performance of the trained model on the test data?

**Comment by violet-zct** [comment]
Create your test data stream and add another DataStreamMonitoring to the extensions.

**Comment by sk210892** [comment]
How many epochs did you usually run to get the results you mentioned and how long did it take?

**Comment by webeng** [comment]
Nice work! Do you have a plot of the training or validation loss per epoch?

Thanks
